doc/tutorials/spike_frequency_adaptation/nestml_spike_frequency_adaptation_tutorial.ipynb

```diff
@@ -0,0 +1,1189 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "NESTML spike frequency adaptation tutorial\n",
+    "==========================================\n",
+    "\n",
+    "Welcome to the NESTML spike frequency adaptation (SFA) tutorial. Here, we will go step by step through adding two different types of SFA mechanism to a neuron model, threshold adaptation and an adaptation current [1], and then evaluate how the new models behave in simulation."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Table of contents\n",
+    "\n",
+    "- [Introduction](#introduction)\n",
+    "- [Threshold-based SFA](#threshold-based-adaptation)\n",
+    "- [Current-based SFA](#current-based-adaptation)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a name=\"introduction\"></a>\n",
+    "\n",
+    "Introduction\n",
+    "------------\n",
+    "\n",
+    "A neuron that is stimulated by rectangular current injections initially responds with a high firing rate, followed by a decrease in the firing rate. This phenomenon is called *spike-frequency adaptation* and is usually mediated by slow K<sup>+</sup> currents, such as a Ca<sup>2+</sup>-activated K<sup>+</sup> current [3]. This behaviour is typically modelled in one of two ways:\n",
+    "\n",
+    "*Threshold adaptation:* The threshold for generating an action potential is increased every time an action potential occurs. The threshold slowly decays back to its original value.\n",
+    "\n",
+    "*Potassium membrane current:* An extra transmembrane current is added to the neuron, that is increased step-wise every time an action potential occurs, and slowly decays back to its original value.\n",
+    "\n",
+    "**Quiz:** If a typical point neuron model has a characteristic timescale (in its membrane potential dynamics) of 20&nbsp;ms, what would be good order-of-magnitude estimate to give to the adaptation characteristic timescale? How does refractoriness figure into this?\n",
+    "\n",
+    "**Hints:** <span style=\"color:white\">Considerably longer than the neuron timescale, e.g. 200 ms; refractoriness defines a maximum neuron firing rate; consider maximum and minimum obtained firing rates in the presence of the SFA mechanism.</span>"
```

Comment:
This isn't visible because the color is set to white. Is this done on purpose?

Reply:
Yes, but it might be a bit silly. I've just made it black.